src/crawlee/events/_event_manager.py

```diff
@@ -68,6 +68,7 @@ def __init__(
 
         Args:
             persist_state_interval: Interval between emitted `PersistState` events to maintain state persistence.
+                The default value is taken from the `Configuration`.
```

Comment:
Same as in `LocalEventManager` - this does not know anything about the Configuration.

Reply:
This is a forgotten docstring update from the previous version. Thanks.